code/__defines/misc.dm

```diff
@@ -51,6 +51,7 @@
 #define STATUS_HUD_OOC 	9 // STATUS_HUD without virus DB check for someone being ill.
 #define LIFE_HUD 		10 // STATUS_HUD that only reports dead or alive
 #define BLINK_HUD       11 // A line representing the blink time remaining on a mob.
+#define PESTILENCE_HUD  12 // A visual indicator of the pestilence.
```

Comment:
Clarify in the comment that it is specifically for 049's "pestilence". I am planning to add the virology eventually and it could be slightly confusing.